### PR TITLE
Better Detection of Emoji in #yelling

### PR DESCRIPTION
**Pull request by bradleysigma**
Now properly detects fake emoji as yelling violations and animated emoji as non-violations.

### CODE REVIEW

**Review by LimaoC — APPROVED** [review]
yeah nah looks about right

File: uqcsbot/yelling.py [hunk 1]
```diff
@@ -154,16 +156,21 @@ def clean_text(self, message: str) -> str:
 
         # ignore emoji and links
         text = re.sub(
-            r":[\w\-\+\~]+:",
+            r"<(?P<animated>a?):(?P<name>[a-zA-Z0-9_]{2,32}):(?P<id>[0-9]{18,22})>",
```
Comment (andrewj-brown) [inline uqcsbot/yelling.py:159]: I'd prefer this to use `\w` and `\d` instead of the character classes `[a-zA-Z0-9_]` and `[0-9]`.

**Review by andrewj-brown — CHANGES_REQUESTED** [review]
regex whinging:tm:

File: uqcsbot/yelling.py (hunk 1 above)
Comment (bradleysigma) [reply uqcsbot/yelling.py]: I just copied the regex from an admin's message in the discordy.py server ¯\_(ツ)_/¯. Anyway, fixed.

**Review by andrewj-brown — COMMENTED** [review]
lgtm

**Review by andrewj-brown — APPROVED** [review]
lgtm 2 (remember to use the radio buttons this time)

**Pull request merged**